bats/1pk5col-ints.bats

```diff
@@ -680,3 +680,44 @@ DELIM
     [ "${#lines[@]}" -eq 6 ]
     [[ "$output" =~ "<NULL>" ]] || false
 }
+
+@test "display correct merge stats" {
+    dolt checkout -b test-branch
+    dolt add test
+    dolt commit -m "added test table"
+    dolt checkout master
+    dolt branch test-branch-m
+    dolt branch test-branch-alt
+    dolt checkout test-branch-m
+    dolt merge test-branch
+    dolt checkout test-branch-alt
+    dolt sql -q "CREATE TABLE test_alt (pk BIGINT NOT NULL COMMENT 'tag:0', c1 BIGINT COMMENT 'tag:1', PRIMARY KEY (pk));"
+    dolt add test_alt
+    dolt commit -m 'add test_alt'
+    dolt checkout test-branch-m
+    dolt merge test-branch-alt
+    dolt add test_alt
+    dolt commit -m 'merge test_alt'
+    dolt checkout test-branch
+    dolt sql -q "insert into test values (0, 1, 2, 3, 4, 5)"
+    dolt add test
+    dolt commit -m "added row to test"
+    dolt checkout test-branch-m
+    run dolt merge test-branch
+    [ "$status" -eq 0 ]
+    [ "${lines[1]}" = "test | 0 " ]
+    skip "Row addition not totalled correctly" [ "${lines[2]}" = "1 tables changed, 1 rows added(+), 0 rows modified(*), 0 rows deleted(-)" ]
+}
+
+@test "checkout table with branch of same name" {
+    dolt checkout -b test
+    dolt add .
+    dolt commit -m "added test table"
+    dolt sql -q "insert into test values (0, 1, 2, 3, 4, 5)"
+    run dolt checkout test
```

Comment:
I believe `git` distinguishes between a branch checkout and a file checkout like so:
`$ git checkout <branch>`
`$ git checkout -- <file>`